src/test/java/catalinazoldi/tests/Tema7/CURAHealthcare.java

```diff
@@ -0,0 +1,115 @@
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+import static org.testng.AssertJUnit.*;
+
+public class CURAHealthcare extends BaseTest {
+
+
+    @DataProvider(name = "data-set")
+    public Object[][] loginDataProvider(){
+        return new Object[][]{
+                {"valid", "John Doe", "ThisIsNotAPassword",},
+                {"invalid", "John Do", "ThisIsNotAPassword",}
+        };
+    }
+
+
+   @Test(dataProvider = "data-set")
+    public void login(String type, String username, String password) {
+        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(2));
+        WebElement menuButton = driver.findElement(By.id("menu-toggle"));
+        menuButton.click();
+        System.out.println("Am dat click pe menu!");
+        WebElement loginButton = driver.findElement(By.xpath("//*[@id=\"sidebar-wrapper\"]/ul/li[3]/a"));
+        loginButton.click();
+        System.out.println("Am facut click pe butonul Login din Menu!");
+        driver.findElement(By.id("txt-username")).sendKeys(username);
+        driver.findElement(By.id("txt-password")).sendKeys(password);
+        WebElement buttonLogin = driver.findElement(By.id("btn-login"));
+        buttonLogin.click();
+        if (type.equals("valid")) {
+            wait.until(ExpectedConditions.presenceOfElementLocated(
+                    By.cssSelector("#appointment > div > div > div > h2")));
+            driver.getCurrentUrl();
+        } else
+            wait.until(ExpectedConditions.presenceOfElementLocated(
+                    By.cssSelector("#login > div > div > div.col-sm-12.text-center > p.lead.text-danger")));
+       // assertEquals(driver.findElement(By.cssSelector("#login > div > div > div.col-sm-12.text-center > p.lead.text-danger")).isDisplayed());
+    }
+
+    @DataProvider(name = "appointmentData")
+    public Object[][] appointmentInfo() {
+        return new Object[][]{
+                {"08.02.2023", "Test", true},
+                {"08.02", "Test", false},
+        };
+    }
+
+    public void completeDetailsForAppointment() {
+        WebElement firstField = driver.findElement(By.id("txt-username"));
+        firstField.sendKeys("John Doe");
+        WebElement secondField = driver.findElement(By.id("txt-password"));
+        secondField.sendKeys("ThisIsNotAPassword");
+        driver.findElement(By.id("btn-login")).click();
+    }
+
+
+    @Test(dataProvider = "appointmentData")
+    public void makeAppointment(String data, String comment, boolean pass) {
+        driver.get("https://katalon-demo-cura.herokuapp.com/");
+        WebElement appointment = driver.findElement(By.id("btn-make-appointment"));
+        appointment.click();
+        completeDetailsForAppointment();
+        Select facility = new Select(driver.findElement(By.id("combo_facility")));
+        facility.selectByVisibleText("Tokyo CURAHealthcare Center");
```

Comment:
Be careful about the option you choose. You're missing a space -> `Tokyo CURA Healthcare Center`